doc/releases/changelog-0.36.0.md

```diff
@@ -31,41 +31,52 @@
 
 <h4>Access an extended arsenal of quantum algorithms 🏹</h4>
 
-* The `FABLE` template is added for efficient block encoding of matrices. Users can now call FABLE to efficiently construct circuits according to a user-set approximation level. 
+* The Fast Approximate BLock-Encodings (FABLE) algorithm for embedding
+  a matrix into a quantum circuit as outlined in
+  [arXiv:2205.00081](https://arxiv.org/abs/2205.00081) is now accessible via the `qml.FABLE` 
+  template.
   [(#5107)](https://github.com/PennyLaneAI/pennylane/pull/5107)
 
-* Create the `qml.Reflection` operator, useful for amplitude amplification and its variants.
-  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
+  The usage of `qml.FABLE` is similar to `qml.BlockEncode` but provides a more
+  efficient circuit construction at the cost of a user-defined approximation 
+  level, `tol`. The number of wires that `qml.FABLE` operates on is `2*n + 1`, 
+  where `n` defines the dimension of the :math:`2^n \times 2^n` matrix that we
+  want to block-encode.
 
   ```python
-  @qml.prod
-  def generator(wires):
-        qml.Hadamard(wires=wires)
+  A = np.array([[0.1, 0.2], [0.3, 0.4]])
+  dev = qml.device('default.qubit', wires=3)
 
-  U = generator(wires=0)
-
-  dev = qml.device('default.qubit')
   @qml.qnode(dev)
   def circuit():
-
-        # Initialize to the state |1>
-        qml.PauliX(wires=0)
-
-        # Apply the reflection
-        qml.Reflection(U)
-
-        return qml.state()
-
-  ```
+      qml.FABLE(A, tol = 0.001, wires=range(3))  
+      return qml.state()
+  ``` 
 
   ```pycon
-  >>> circuit()
-  tensor([1.+6.123234e-17j, 0.-6.123234e-17j], requires_grad=True)
+  >>> mat = qml.matrix(circuit)()
+  >>> 2 * mat[0:2, 0:2]
+  array([[0.1+0.j, 0.2+0.j],
+         [0.3+0.j, 0.4+0.j]])
   ```
-  
-* The `qml.AmplitudeAmplification` operator is introduced, which is a high-level interface for amplitude amplification and its variants.
+
+* A high-level interface for amplitude amplification and its variants is now 
+  available via the new `qml.AmplitudeAmplification` template.
   [(#5160)](https://github.com/PennyLaneAI/pennylane/pull/5160)
 
+  Based on [arXiv:quant-ph/0005055](https://arxiv.org/abs/quant-ph/0005055), 
+  given a state :math:`\vert \Psi \rangle = \alpha \vert \phi \rangle + \beta \vert \phi^{\perp} \rangle`, 
+  `qml.AmplitudeAmplification` amplifies the amplitude of :math:`\vert \phi \rangle`. 
+  Mathematically, 
+  :math:`\texttt{AmplitudeAmplification} \vert \Psi \rangle \approx \vert \phi \rangle`.
+
+  Here's an example with 
```

Comment:
```suggestion
  Here's an example with a target state
```

Reply:
Just adding these suggestions because we took at the added information about `U` and `O`.